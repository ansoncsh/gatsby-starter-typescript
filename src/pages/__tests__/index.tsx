import * as React from "react"
import * as renderer from "react-test-renderer"
import IndexPage from "../index"

describe("IndexPage", () => {
    it("renders correctly", () => {
        const data = {
            site: {
                siteMetadata: {
                    title: `Gatsby Typescript Starter`,
                },
            },
        }
        const tree = renderer.create(<IndexPage data={data} />).toJSON()
        expect(tree).toMatchSnapshot()
    })
})
