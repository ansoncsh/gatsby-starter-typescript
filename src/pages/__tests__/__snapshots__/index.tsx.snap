// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`IndexPage renders correctly 1`] = `
<div>
  <h1>
    Hi people
  </h1>
  <p>
    Welcome to your new
     
    <strong>
      Gatsby Typescript Starter
    </strong>
     site.
  </p>
  <p>
    Now go build something great.
  </p>
  <a
    href="/about/"
    onClick={[Function]}
    onMouseEnter={[Function]}
  >
    Go to about page
  </a>
</div>
`
